src/d_main.cpp

```diff
@@ -684,8 +684,19 @@ void D_Display ()
 
 	if (viewactive)
 	{
-		R_SetFOV (r_viewpoint, players[consoleplayer].camera && players[consoleplayer].camera->player ?
-			players[consoleplayer].camera->player->FOV : players[consoleplayer].camera->CameraFOV);
+		DAngle fov = 90.f;
+		if (players[consoleplayer].camera != nullptr)
+		{
+			if (players[consoleplayer].camera && players[consoleplayer].camera->player)
```

Comment:
players[consoleplayer].camera does not need to be checked twice. This is redundant.

However, players[consoleplayer].camera->player does need to be checked, so it'll be good to leave that there.

Otherwise, this looks good to me.

Reply:
Probably I didn't mention this clear enough but my initial suggestion was to assign `players[consoleplayer].camera` to a local variable so all those checks will be much easier to read.

Reply:
Okay, done. Sorry about that, I wasn't sure what the policy was on making code as short as possible, so I tried to preserve Randi's work back there. I've shortened it now.